### PR TITLE
refactor(components): refactor roundtab stories

### PR DESCRIPTION
**Pull request by koji**
update roundtab stories to align with the actual app design

close AUTH-

# Overview
update RoundTab component stories to align with the actual app design

2024-04-19 00_43_39-Library _ Molecules _ RoundTab - Round Tab ⋅ Storybook and 85 more pages - Perso

close AUTH-358

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: components/src/molecules/RoundTab.stories.tsx [hunk 1]
```diff
@@ -1,55 +1,80 @@
-import * as React from 'react'
+import React from 'react'
```
Comment (jerader) [inline components/src/molecules/RoundTab.stories.tsx]: why did you have to change this?

**Review by jerader — APPROVED** [review]
lgtm! left one comment

File: components/src/molecules/RoundTab.stories.tsx (hunk 1 above)
Comment (koji) [reply components/src/molecules/RoundTab.stories.tsx]: noticed this
https://github.com/Opentrons/opentrons/blob/edge/tsconfig-base.json#L9

Comment (koji) [reply components/src/molecules/RoundTab.stories.tsx]: and slightly smaller than * as React. (I'm using React for opentrons-ai-client)
I will use `* as React` for this moment.

**Pull request merged**